include/sbat_var_defs.h

```diff
@@ -22,14 +25,14 @@
 	SBAT_VAR_SIG SBAT_VAR_VERSION SBAT_VAR_LATEST_DATE "\n" \
 	SBAT_VAR_LATEST_REVOCATIONS
 #else /* !ENABLE_SHIM_DEVEL */
-#define SBAT_VAR_PREVIOUS_DATE SBAT_VAR_ORIGINAL_DATE
-#define SBAT_VAR_PREVIOUS_REVOCATIONS
+#define SBAT_VAR_PREVIOUS_DATE "2022052400"
+#define SBAT_VAR_PREVIOUS_REVOCATIONS "shim,2\ngrub,2\n"
```

Comment:
We said we don't bump previous to shim,2 because most folks (including Ubuntu, SUSE, openSUSE) don't have a shim,2 yet and would end up unbootable.

Reply:
Maybe add a comment stating so above it.

Reply:
So you're suggesting `"grub,2\n"`, or `"shim1,\ngrub,2\n"`?

Reply:
I only tested `"grub,2\n"` (and @jsetje had the same in the end) and we didn't have `shim,1\ngrub,1\n` before so the long one would be a bit inconsistent.

Reply:
They are functionally equivalent, but "grub,2\n" is the more compact way of stating it.

Reply:
Fixed in https://github.com/rhboot/shim/pull/528/commits/f38e63aa61deb926d51cf07d05a7a2e140d1be58.